cluster/images/conformance/Makefile

```diff
@@ -54,9 +54,10 @@ endif
 	chmod a+rx ${TEMP_DIR}/e2e.test
 	chmod a+rx ${TEMP_DIR}/gorunner
 
-	cd ${TEMP_DIR} && sed -i.back "s|BASEIMAGE|${BASEIMAGE}|g" Dockerfile
-
-	docker build --pull -t ${REGISTRY}/conformance-${ARCH}:${VERSION} ${TEMP_DIR}
+	docker rmi ${BASEIMAGE} 2>/dev/null || :
```

Comment:
went ahead and did a quick skim: i'm pretty sure this should be safe without rmi when using buildx fwiw...
we have other similar builds that do not do this.
removing the image is kinda bad behavior, it breaks e.g. offline building.
currently you can build kubernetes offline with little trouble.